pages/blog/[slug].js

```diff
@@ -1,11 +1,32 @@
 import { Layout } from '../../components/layout';
 import { Container } from '../../components/container';
 import { getPost, getSlugsOfPosts } from '../../api/blog';
+import ReactMarkdown from 'react-markdown';
+import { Code } from '../../components/code';
 
 const BlogPost = ({ page, menu, footer }) => {
   return (
     <Layout menu={menu.menuItems} footer={footer}>
-      <Container>Blog Post</Container>
+      <Container>
+        <div key={page.key}>
+          <img src={page.banner} />
+          <h1>{page.title}</h1>
+          <div>{page.publish_date}</div>
+          <div>
+            {page.blocks.map((block) => (
+              <div key={block.id} title={block.title}>
```

Comment:
there is as misunderstanding here title should be displayed as an element in the ui not a hover over action. Therefore it needs a dedicated block. for instance
```
<div className="block-title">{block.title}</div>
```
and this block can be placed in an if case to prevent the ui breaking on non-existent title in the block object